tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/lldb-settings.sh

```diff
@@ -103,3 +103,7 @@ then
 settings append target.source-map ./external/ "$BAZEL_EXTERNAL_DIRNAME"
 END
 fi
+
+# Xcode 15.x: Fixes a problem in the flag parser of LLDB 15.1 potentially causing LLDB to crash
+sed -i '' 's, -vfsoverlay, -vfsoverlay ,g' $BAZEL_LLDB_INIT_FILE
```

Comment:
Is there a concern that this causes issues with indexing?

Reply:
>Is there a concern that this causes issues with indexing?

I tested basic indexing stuff after applying this and it was fine, but I do want to run a more complete set of manual tests before calling it "safe"

>Should we conditionally set this for Xcode 15.1?

Good callout, yes we should

Reply:
This is not the final approach btw, once I remove it from draft I'll ask for new reviews. Ignore what's in here for now.